tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/bin/pydoc: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-tk/test/test_tkinter/test_images.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-tk/test/test_tkinter/test_loadtk.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-tk/test/README: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-tk/Canvas.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-tk/Tix.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/easy-install.pth: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/selector.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/commands.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/sphinx/pluginopts.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/plugins/logcapture.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/testing/tests/test_doctesting.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/polynomial/tests/test_legendre.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/polynomial/tests/test_polynomial.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/polynomial/tests/test_classes.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/polynomial/tests/test_laguerre.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/polynomial/tests/test_hermite_e.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/matrixlib/setup.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/f2py/cb_rules.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/f2py/common_rules.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/f2py/tests/test_callback.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/doc/indexing.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/doc/structured_arrays.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/fcompiler/sun.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/fcompiler/compaq.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/fcompiler/ibm.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/fcompiler/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/command/build_ext.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/command/bdist_rpm.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/intelccompiler.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/tests/swig_ext/src/zoo.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/distutils/tests/f2py_ext/tests/test_fib2.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/_internal.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/fromnumeric.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/tests/test_numerictypes.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/tests/test_print.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/tests/test_scalarprint.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/tests/test_defchararray.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/tests/test_multiarray_assignment.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/numpy/core/lib/npy-pkg-config/npymath.ini: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/_lib/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/_lib/six.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/_lib/tests/test_tmpdirs.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/_lib/tests/test__version.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/cpp_namespace_spec.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/swigptr.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/swig2_spec.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/slice_handler.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/base_info.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/array.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/matsymm.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/matgen.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/applics.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/extremum.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/Makefile.am: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/generate/genpromote.cpp: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/generate/genmatbops.cpp: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/generate/genvecbops.cpp: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/generate/genvecbfn.cpp: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/array/resize.cc: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/array/reduce.cc: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/array/cycle.cc: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/array/funcs.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/array/functorExpr.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/meta/metaprog.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/meta/matvec.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/meta/vecassign.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/meta/Makefile.am: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/blitz/blitz/meta/Makefile.am: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/weave/tests/test_blitz_tools.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/ndimage/setup.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/ndimage/_ni_support.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/stats/setup.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/stats/mstats_basic.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/stats/stats.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/special/_ufuncs.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/special/tests/test_data.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/special/tests/test_ellip_harm.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/spatial/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/spatial/tests/data/pdist-hamming-ml.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/spatial/tests/data/pdist-correlation-ml.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/spatial/tests/data/cdist-X2.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/csgraph/_reordering.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/linalg/isolve/lsqr.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/linalg/isolve/tests/test_utils.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/generate_sparsetools.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/sparsetools.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/data.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/sparse/coo.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/signal/_spectral.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/signal/ltisys.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/signal/fir_filter_design.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/signal/_arraytools.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/signal/tests/test_filter_design.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/optimize/_slsqp.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/optimize/_lbfgsb.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/optimize/_differentialevolution.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/optimize/_trustregion.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/optimize/tests/test_linprog.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/misc/setup.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/misc/lena.dat: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/_solve_toeplitz.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/_fblas.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/decomp_svd.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/decomp_cholesky.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/_matfuncs_sqrtm.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/linalg_version.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/tests/test_lapack.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/tests/test_basic.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/linalg/tests/test_solvers.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/io/harwell_boeing/hb.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/io/matlab/streams.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/interpolate/tests/test_ndgriddata.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/integrate/_test_multivariate.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/integrate/tests/test_quadrature.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/cluster/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/setup.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/HACKING.rst.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/scipy/INSTALL.rst.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/mpi4py/lib-pmpi/libvt.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/mpi4py/lib-pmpi/libmpe.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/mpi4py/include/mpi4py/mpi4py.MPI_api.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/mpi4py/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/EGG-INFO/top_level.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/EGG-INFO/dependency_links.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/StringIOTree.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/TestUtils.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/Code.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/FlowControl.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/Future.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/ParseTreeTransforms.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/TreeFragment.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/Version.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/Version.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/FlowControl.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/Scanning.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Compiler/Visitor.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Plex/Errors.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Plex/Machines.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Utility/Embed.c: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Utility/ExtensionTypes.c: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Includes/cpython/long.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Includes/cpython/mapping.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Includes/cpython/version.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Includes/libcpp/vector.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/Cython/Includes/posix/wait.pxd: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/pyximport/pyxbuild.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/Cython-0.23.4-py2.7-linux-x86_64.egg/pyximport/pyximport.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/_cntr.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/container.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backend_bases.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/_pylab_helpers.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/cm.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/hatch.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/finance.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/cbook.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/_path.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backends/backend_cocoaagg.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backends/backend_gtk3.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backends/qt_editor/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backends/web_backend/mpl_tornado.js: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backends/web_backend/jquery/css/themes/base/jquery-ui.min.css: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backends/Matplotlib.nib/keyedobjects.nib: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/backends/Matplotlib.nib/classes.nib: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/axes/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/sphinxext/tests/tinypages/_static/README.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/testing/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/testing/jpl_units/UnitDblConverter.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/testing/jpl_units/Duration.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/delaunay/interpolate.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/externals/six.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/externals/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/test_texmanager.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/test_arrow_patches.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/test_colorbar.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/test_image.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/test_mlab.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/test_labeled_data_unpacking.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_legend/scatter_rc1.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_legend/legend_auto3.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_legend/scatter_rc3.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_legend/scatter_rc1.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_legend/legend_various_labels.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_10.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_73.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_18.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_27.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_14.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_66.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_55.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_19.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_20.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_20.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_69.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_50.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_32.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_45.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_32.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_32.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_06.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_45.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_11.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_28.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_27.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_15.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_49.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_07.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_60.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_41.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_59.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_01.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_48.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_16.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_40.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_29.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_72.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_71.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_50.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_61.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_55.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_16.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_26.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_64.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_64.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_47.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_18.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_31.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_75.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_04.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_12.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_11.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_06.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_77.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_29.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_09.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_11.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_34.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_17.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_68.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_56.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_59.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_12.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_00.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_dejavuserif_79.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_30.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_54.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_26.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stix_10.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_cm_53.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_17.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_22.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_mathtext/mathtext_stixsans_74.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_colorbar/colorbar_closed_patch.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_patheffects/patheffect1.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_simplification/hatch_simplify.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_simplification/clipper_edge.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_contour/contour_manual_labels.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_streamplot/streamplot_colormap_test_image.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_subplots/subplots_offset_text.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_transforms/pre_transform_data.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_tightlayout/tight_layout_offsetboxes2.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_tightlayout/tight_layout7.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_tightlayout/tight_layout2.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_tightlayout/tight_layout_offsetboxes1.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_tightlayout/tight_layout8.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_tightlayout/tight_layout6.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_tightlayout/tight_layout5.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_text/titles.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_text/multiline.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/stackplot_test_image.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/polar_units.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/mollweide_grid.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/markevery.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/hist_stacked_normed.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/autoscale_tiny_range.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/hist_stacked_weights.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/boxplot.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/symlog.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/errorbar_basic.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/formatter_ticker_004.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/units_strings.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_axes/hist_steplog.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_spines/spines_data_positions.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_patches/wedge_range.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_patches/patch_custom_linestyle.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_patches/patch_alpha_coloring.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__set_color.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__add_positions.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__set_linestyle.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__switch_orientation.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__set_linestyle.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__set_linewidth.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__set_positions.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/tests/baseline_images/test_collections/EventCollection_plot__set_lineoffset.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/matplotlibrc: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/images/forward.xpm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/images/stock_close.xpm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/images/forward.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/images/zoom_to_rect.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/images/matplotlib.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/images/move.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/ttf/STIXGeneralBolIta.ttf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/ttf/STIXNonUniBol.ttf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/ttf/VeraMono.ttf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/ttf/VeraMoIt.ttf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/pdfcorefonts/ZapfDingbats.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/pdfcorefonts/Courier-Oblique.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/pdfcorefonts/Helvetica-BoldOblique.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/pdfcorefonts/Times-Bold.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/phvbo8an.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/putb8a.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/pplri8a.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/pagk8a.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/cmex10.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/pcrbo8a.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/putbi8a.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/fonts/afm/pzcmi8a.afm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/matplotlib/mpl-data/sample_data/jacksboro_fault_dem.npz: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/axes_grid1/axes_size.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/axisartist/axis_artist.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/tests/baseline_images/test_mplot3d/bar3d.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/tests/baseline_images/test_mplot3d/bar3d.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/tests/baseline_images/test_mplot3d/contour3d.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/tests/baseline_images/test_mplot3d/lines3d.pdf: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/matplotlib-1.5.1-py2.7-linux-x86_64.egg/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_masked.svg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/python_dateutil-2.5.3-py2.7.egg: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/compat/pickle_compat.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/computation/common.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/computation/align.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/core/style.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/core/series.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/core/reshape.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/core/panel4d.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/core/categorical.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/core/api.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/stata.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/gbq.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/ga.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/date_converters.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/common.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/test_sas.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/test_pickle.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/test_gbq.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/test_common.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/generate_legacy_storage_files.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/banklist.csv: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/SSHSV1_A.XPT: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/stata9_117.dta: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/stata2_117.dta: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/times_1900.xls: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/test_converters.xls: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/test2.xls: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/test1.xls: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/blank.xls: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/blank.xlsm: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_hdf/legacy_table_0.11.h5: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_hdf/legacy_table_0.11.h5: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_hdf/legacy.h5: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_pickle/0.16.2/0.16.2_x86_64_darwin_3.4.3.pickle: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_pickle/0.16.2/0.16.2_x86_64_darwin_2.7.10.pickle: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_pickle/0.13.0/0.13.0_x86_64_linux_3.3.0.pickle: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_pickle/0.13.0/0.13.0_x86_64_linux_2.7.8.pickle: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_pickle/0.11.0/x86_64_linux_3.3.0.pickle: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_pickle/0.11.0/x86_64_linux_2.7.3.pickle: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_msgpack/0.16.2/0.16.2_x86_64_linux_3.4.3.msgpack: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/io/tests/data/legacy_msgpack/0.16.2/0.16.2_AMD64_windows_3.4.3.msgpack: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/stats/ols.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/stats/math.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/stats/tests/test_var.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/util/terminal.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tests/test_strings.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tests/test_panelnd.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tests/test_lib.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tests/test_groupby.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tests/test_format.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tests/test_expressions.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tests/test_base.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tools/tests/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tseries/period.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tseries/interval.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tseries/base.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tseries/tests/test_timeseries_legacy.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tseries/tests/test_period.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pandas-0.17.1-py2.7-linux-x86_64.egg/pandas/tseries/tests/test_daterange.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/locations.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/utils/ui.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/commands/search.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/commands/download.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/commands/uninstall.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/commands/install.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/commands/help.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/colorama/win32.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/packaging/requirements.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/html5lib/utils.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/html5lib/filters/whitespace.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/html5lib/filters/_base.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/html5lib/treebuilders/etree_lxml.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/distlib/manifest.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/distlib/t32.exe: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/compat.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/structures.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/chardet/langbulgarianmodel.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/chardet/euctwprober.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/chardet/compat.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/chardet/chardistribution.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/chardet/utf8prober.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/chardet/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/request.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/request.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/packages/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/packages/ssl_match_hostname/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/contrib/pyopenssl.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/fork_wait.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/ieee754.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test___future__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_abc.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_asynchat.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_atexit.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_codecmaps_jp.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_copy_reg.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_csv.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_distutils.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_dummy_thread.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_email.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_ftplib.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_imp.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_logging.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_macos.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_macpath.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_minidom.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_module.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_ossaudiodev.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_pty.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_py_compile.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_quopri.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_setcomps.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_sha.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_socketserver.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_sort.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_spwd.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_stringprep.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_sunau.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_sundry.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_timeout.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_transformer.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_xml_etree_c.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_zipimport_support.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/testtar.tar: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/audiodata/pluck-pcm32.wav: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/audiodata/pluck-ulaw.au: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/capath/4e1295a3.0: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/ddCanonical.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/ddCompareTotal.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/ddDivideInt.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/ddScaleB.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/decDouble.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/dqCopyAbs.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/dqNextToward.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/dqReduce.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/dqShift.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/exp.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/or.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/remainderNear.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/rescale.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/decimaltestdata/rotate.decTest: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/buffer_tests.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/mapping_tests.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/pydoc_mod.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/reperf.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/revocation.crl: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/sample_doctest_no_docstrings.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/script_helper.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/seq_tests.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_MimeWriter.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_SimpleHTTPServer.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test__locale.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_base64.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_difflib.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_epoll.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_file.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_io.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_lib2to3.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_linuxaudiodev.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_multifile.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_nis.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_parser.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_pep247.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_pep277.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_random.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_repr.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_slice.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_startfile.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_str.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_support.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_sys_settrace.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/test/test_threading_local.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/cp1006.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/cp857.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/cp861.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/cp864.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/cp865.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/cp865.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/iso8859_15.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/mac_roman.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/mac_romanian.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/encodings/ptcp154.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/email/mime/application.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/email/test/data/msg_14.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/email/test/data/msg_18.txt: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/email/test/test_email_renamed.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/email/iterators.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/email/quoprimime.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/ensurepip/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/sqlite3/test/dump.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/sqlite3/test/hooks.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/sqlite3/test/regression.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/bsddb/db.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/importlib/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/wsgiref/util.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib2to3/fixes/fix_map.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib2to3/fixes/fix_throw.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib2to3/pgen2/driver.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib2to3/tests/data/fixers/myfixes/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib2to3/tests/data/fixers/myfixes/fix_preorder.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib2to3/tests/data/fixers/parrot_example.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib2to3/fixer_util.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/ctypes/test/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/ctypes/test/test_bitfields.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/Icons/idle_32.gif: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/idle_test/test_parenmatch.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/FileList.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/FormatParagraph.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/IdleHistory.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/PyShell.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/RemoteDebugger.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/ToolTip.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/config-highlight.def: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/idle.bat: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/idle.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/run.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/idlelib/run.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/command/build_py.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/command/check.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/command/command_template: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/command/config.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/command/install_headers.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/tests/setuptools_build_ext.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/tests/setuptools_extension.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/tests/test_install.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/cygwinccompiler.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/filelist.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/msvccompiler.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/text_file.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/distutils/versionpredicate.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/xml/dom/domreg.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/xml/dom/minicompat.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/xml/etree/ElementTree.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/multiprocessing/sharedctypes.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/unittest/test/test_skipping.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/unittest/signals.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/unittest/util.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/curses/__init__.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/plat-linux2/IN.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/plat-linux2/TYPES.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/ConfigParser.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/MimeWriter.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/Queue.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/_LWPCookieJar.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/_osx_support.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/_pyio.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/asynchat.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/audiodev.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/bisect.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/cgi.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/cgitb.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/chunk.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/collections.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/dumbdbm.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/functools.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/heapq.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/httplib.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/py_compile.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/pydoc.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/shutil.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/webbrowser.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/xmllib.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/zipfile.py: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/wsgiref.egg-info: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-dynload/_elementtree.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-dynload/_sqlite3.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-dynload/grp.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-dynload/unicodedata.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-dynload/_functools.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-dynload/future_builtins.so: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/lib/python2.7/lib-dynload/strop.so: Cannot open: File exists
tar: Exiting with failure status due to previous errors
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/include/python2.7/py_curses.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/include/python2.7/pycapsule.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/include/python2.7/structmember.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/include/python2.7/structseq.h: Cannot open: File exists
tar: apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/include/python2.7/sysmodule.h: Cannot open: File exists
tar: Exiting with failure status due to previous errors
tar: Exiting with failure status due to previous errors
tar: Exiting with failure status due to previous errors
python=/dev/shm/piccinal/apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/bin/python
python=/dev/shm/piccinal/apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/bin/python
start
start
python=/dev/shm/piccinal/apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/bin/python
start
python=/dev/shm/piccinal/apps/dora/UES/5.2.UP04/broadwell/easybuild/software/Python/2.7.11-CrayGNU-2016.03/bin/python
start
time to import numpy t=0.470294 rk=0 numpy=1.10.4
time to import numpy t=0.470471 rk=1 numpy=1.10.4
time to import numpy t=0.470607 rk=3 numpy=1.10.4
time to import numpy t=0.470938 rk=2 numpy=1.10.4
end
end
end
end
real 9.32
user 0.00
sys 0.11
